File: app/src/main/java/com/akexorcist/checkscreen/MainActivity.java
package com.akexorcist.checkscreen;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;

import com.akexorcist.screenchecker.ScreenCheckerActivity;

public class MainActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        startActivity(new Intent(this, ScreenCheckerActivity.class));
        finish();
    }
}

File: app/src/main/java/app/akexorcist/checkscreen/MainActivity.kt
package app.akexorcist.checkscreen

import android.content.Intent
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.akexorcist.screenchecker.ScreenCheckerActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, ScreenCheckerActivity::class.java))
        finish()
    }
}
